panda-utilities/src/test/java/org/panda_lang/utilities/commons/iterable/ArrayIterableTest.java

```diff
@@ -0,0 +1,26 @@
+package org.panda_lang.utilities.commons.iterable;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Iterator;
+
+class ArrayIterableTest {
+
+    @Test
+    void testArrayIterable() {
+        ArrayIterable<String> arrayIterable =
+                new ArrayIterable<>(new String[]{"a", "b", "c"});
+
+
```

Comment:
Tests contains a lot of doubled empty lines, there is no need to keep it that way. 
Keep declarations together or group them in related context.

Reply:
Please check the commit bee18e9 to make sure it's fixed.